client/my-sites/activity/activity-log/style.scss

```diff
@@ -27,15 +26,15 @@
 		bottom: 0;
 		height: 16px;
 		z-index: -1;
-		background: blue;
-		background: linear-gradient( transparentize( $gray-light, 1 ), $gray-light );
+		background: var( --color-neutral-100 );
```

Comment:
no idea why this was blue. It's supposed to be a little gradient to soften the end of the activity timeline line.